code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -67,14 +66,24 @@
 
 	return the_id
 
-/datum/reagents/proc/update_total() // Updates volume.
+/datum/reagents/proc/update_total() // Updates volume and temperature. 3 passes.
+
 	total_volume = 0
+
 	for(var/datum/reagent/R in reagent_list)
 		if(R.volume < MINIMUM_CHEMICAL_VOLUME)
 			del_reagent(R.id)
 		else
 			total_volume += R.volume
-	return
+
+	return total_volume > 0

```

Comment:
Might be worth returning `max(0, total_volume)` instead. This comparison seems very constrained to a specific use.